website/docs/reference/resource-configs/snowflake-configs.md

```diff
@@ -678,3 +678,27 @@ Per the [Snowflake documentation](https://docs.snowflake.com/en/sql-reference/in
   >- DDL operations.
   >- DML operations (for tables only).
   >- Background maintenance operations on metadata performed by Snowflake.
+
+<VersionBlock firstVersion="1.9">
+
+## Pagination for object results
+
+By default, when dbt encounters a schema with up to 100k objects, it will paginate the results from `show objects` at 10k per page for up to 10 pages.
```

Comment:
```suggestion
By default, when dbt encounters a schema with up to 100,000 objects, it will paginate the results from `show objects` at 10,000 per page for up to 10 pages.
```